mssql-plugin/widgets/SqlServer-action.json

```diff
@@ -85,6 +82,106 @@
           "widget-attributes": {
             "default": "-1"
           }
+        },
+        {
+          "label": "Application Intent",
+          "name": "applicationIntent",
+          "widget-type": "select",
```

Comment:
I think an inline radio-group would be better than select, since there are only 2 options but it doesn't really fit a toggle.

Reply:
Changed.